packages/backend/src/api/resources/AttributeMapping.ts

```diff
@@ -0,0 +1,14 @@
+import type { AttributeMappingJSON } from './JSON';
```

Comment:
❓ Maybe we should move this in the saml connection resource file instead as it's part of it and not an different resource. Not sure how we handle these cases